src/consumer/consumerGroup.js

```diff
@@ -187,7 +187,6 @@ module.exports = class ConsumerGroup {
     this.leaderId = groupData.leaderId
     this.memberId = groupData.memberId
     this.members = groupData.members
-    this.groupInstanceId = groupData.groupInstanceId
```

Comment:
the response from broker is null, hence groupInstanceId is overwritten and member becomes a dynamic one.